cluster/vagrant/util.sh

```diff
@@ -132,13 +132,27 @@ function create-provision-scripts {
   for (( i=0; i<${#NODE_NAMES[@]}; i++)); do
     (
       echo "#! /bin/bash"
-      echo-kube-env
-      echo "NODE_NAME=(${NODE_NAMES[$i]})"
-      echo "NODE_IP='${NODE_IPS[$i]}'"
-      echo "NODE_ID='$i'"
-      echo "CONTAINER_ADDR='${NODE_CONTAINER_ADDRS[$i]}'"
-      echo "CONTAINER_NETMASK='${NODE_CONTAINER_NETMASKS[$i]}'"
-      awk '!/^#/' "${KUBE_ROOT}/cluster/vagrant/provision-utils.sh"
+      echo "MASTER_NAME='${MASTER_NAME}'"
+      echo "MASTER_IP='${MASTER_IP}'"
+      echo "MINION_NAMES=(${MINION_NAMES[@]})"
```

Comment:
this should be `NODE_NAMES`
All other instances of MINION should now be NODE, but these first few are breaking.